mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanrepaymentschedule/LoanRepaymentSchedulePresenter.java

```diff
@@ -6,18 +6,18 @@
 
 import javax.inject.Inject;
 
-import rx.Subscriber;
-import rx.Subscription;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.observers.DisposableObserver;
+import io.reactivex.schedulers.Schedulers;
 
 /**
  * Created by Rajan Maurya on 08/06/16.
  */
 public class LoanRepaymentSchedulePresenter extends BasePresenter<LoanRepaymentScheduleMvpView> {
 
     private final DataManager mDataManager;
-    private Subscription mSubscription;
+    private CompositeDisposable compositeDisposable;
```

Comment:
@therajanmaurya I'm a bit confused about this. Does `Subscription` will be replaced by `CompositeDisposable` ? Because I'm having problems in this type of changes.

Reply:
@Rish02Sharma 

```
CompositeDisposable disposables = new CompositeDisposable();
and then you can add your observables 
disposables.add(mDataManager.getLoanRepaySchedule(loanId))
```